components/feature/addons/src/main/java/mozilla/components/feature/addons/migration/SupportedAddonsChecker.kt

```diff
@@ -154,6 +157,12 @@ internal class SupportedAddonsWorker(
                 addon.isSupported() && addon.isDisabledAsUnsupported()
             }
 
+            withContext(Dispatchers.Main) {
+                newSupportedAddons.map {
+                    addonManager.enableAddon(it, source = EnableSource.APP_SUPPORT)
```

Comment:
Let's pass an error callback and log to Sentry? This way we know if we failed to enable? Wdyt?

Reply:
Agree!